### ISSUE TITLE
Neptune + detectron2 Integration

### ISSUE THREAD

**Issue by kshitij12345** [facebookresearch/detectron2#4677]
Hi,

Kshiteej Kalambarkar here, a Developer Advocate at [Neptune.ai](https://www.neptune.ai/)

I'm working on the detectron2 integration to help detectron2 users easily log their experiment's metadata to Neptune with a few lines of code.

I would like to connect with you (maintainers) and know more about your product roadmap (i.e. upcoming features and breaking changes) and if you'd be interested in upstreaming it into the detectron2 repository.

Thank you

**Comment by Robotatron** [comment]
Yes, please, as a user, I am interested to see Neptupe being integrated

**Comment by okomenchuk** [comment]
> Hi,
> 
> Kshiteej Kalambarkar here, a Developer Advocate at [Neptune.ai](https://www.neptune.ai/)
> 
> I'm working on the detectron2 integration to help detectron2 users easily log their experiment's metadata to Neptune with a few lines of code.
> 
> I would like to connect with you (maintainers) and know more about your product roadmap (i.e. upcoming features and breaking changes) and if you'd be interested in upstreaming it into the detectron2 repository.
> 
> Thank you

Hello! Do you have any updates on this?

**Comment by kshitij12345** [comment]
Hi @okomenchuk,

You can have a look at https://docs.neptune.ai/integrations/detectron2/
